include/swift/SIL/SILModule.h

```diff
@@ -459,6 +459,8 @@ class SILModule {
   /// Link in all VTables in the module.
   void linkAllVTables();
 
+  void linkAll();
```

Comment:
I'm going to redo this and add comments etc. I just wanted to get feedback on the general approach(and it turns out I forgot WMO exists)